src/js/modals/CodeSnippetModal.jsx

```diff
@@ -37,6 +38,10 @@ const MAPZENJS_FULL_SNIPPET = `<!DOCTYPE html>
           scene: '{{ scene }}'
         }
       });
+      L.control.scale().addTo(map);
```

Comment:
I'm somewhat skeptical of adding the Leaflet scale to every Play map?

Reply:
Fair point. Could go either way, but I was pleasantly surprised at how good it looked. 

It is a question I've seen asked fairly often, especially from beginners. Maybe leave it in there but comment it out?